### DISCUSSION TITLE
"Any crashes are potentially security vulnerabilities and should be treated as such."

### DISCUSSION THREAD

**Discussion by skissane-medallia** [istio/istio#54078] [Q&A] [answered]
When I go to log an issue, it says this. But does that include `istioctl`? I've found a bug in `istioctl` where, given a certain configuration problem, a command fails with a `SIGSEGV` instead of printing a meaningful error. But I'm sceptical this could be exploitable as a security vulnerability in any way, even though I suppose a `SIGSEGV` technically a "crash". I assume that message is really talking about the istio runtime not the CLI, since I can see how a crash in the runtime could easily be a security vulnerability. But I'm not sure because the message isn't explicit about excluding the CLI.

**Comment by howardjohn** [comment, marked as answer]
Its unlikely to, but in theory could. Erroring on the side of caution it would be best to send it there, though the likely response would be to open a public GitHub issue. Always better to have an issue privately reported that isn't a security issue, than one that _is_ but is reported publicly.

**Reply by skissane-medallia** [reply]
Thanks for the feedback. I sent an email to istio-security-vulnerability-reports@googlegroups.com with details of the issue I found
